exonum/src/blockchain/tests.rs

```diff
@@ -61,16 +62,16 @@ fn test_u64() {
 
 #[test]
 fn test_system_time() {
-    use std::time::{SystemTime, UNIX_EPOCH};
+    use std::time::UNIX_EPOCH;
     encoding_struct! {
         struct Test {
-            some_test: SystemTime,
+            some_test: DateTime<Utc>,
         }
     }
     let test_data = r##"{"some_test":{"nanos":0,"secs":"0"}}"##;
 
 
-    let test = Test::new(UNIX_EPOCH);
+    let test = Test::new(UNIX_EPOCH.into());
```

Comment:
Are there any advantages of using `UNIX_EPOCH` here instead of `Utc.timestamp(0, 0)`?

Reply:
Possibly provide user some example how to use `SystemTime` in messages? I dunno actually, will change.